unit6/bai2/style.css

```
@@ -42,4 +42,17 @@ label input {
 }
   :target {
 	background: #fac7c3;
+  }
+  input#avatar {
+	  display: none;
+  }
+  .up_avatar {
+	  width: 50px;
+	  height: 50px;
+	  background: #fff;
+	  border-radius: 50%;
+	  float: left;
+  }
+  input[type="radio"]:checked+label{
```

```suggestion
  input[type="radio"]:checked + label {
```
Nhìn kỹ cách đặt dấu cách